text/0000-remove-application-wrapper.md

```diff
@@ -0,0 +1,97 @@
+- Start Date: 2017-12-11
+- RFC PR: https://github.com/emberjs/rfcs/pull/280
+- Ember Issue: (leave this empty)
+
+# Summary
+
+Introduce a low-level "flag" to remove the automatic wrapper `<div>` around
+Ember apps and tests.
+
+# Motivation
+
+In Ember applications today, applications are anchored to some existing HTML
+element in the page. Usually, this element is the `<body>` of the document, but it
+can be configured to be a different one when the application is defined,
+passing a CSS selector to the `rootElement` property:
+
+```js
+export default Ember.Application.extend({
+  rootElement: '#app'
+});
+```
+
+However, whatever the root is, the application adds another `<div>` wrapper
+that is not required anymore. It's a vestigial remainder of some implementation
+detail of how views worked in Ember 1.x. Some sort of wisdom tooth of the original
+rendering system that serves no purpose today.
+
+Furthermore, much like a wisdom tooth, it can give us problems. In the past, this element
+was configurable using the `ApplicationView`, but when views were removed we lost that
+ability. Right now we are stuck with a wrapper element we can't remove nor customize,
+which is why some apps target the selector `body > .ember-view` to style this element.
+
+Similarly, in testing there is another `.ember-view` wrapper inside the
+`#ember-testing` container for no good reason.
+
+This RFC proposes to add a global flag to remove those wrapper elements,
+effectively making the `application.hbs` template have "Outer HTML" semantics, which aligns
+well with [the changes recently proposed](#278) for template-only components,
```

Comment:
The link to #278 is broken here (lmk if it was left intentionally I'm gonna remove this comment)

Reply:
Good catch, fixed!